packages/types/src/types/cloudinary-video-player.ts

```diff
@@ -495,4 +499,343 @@ export interface CloudinaryVideoPlayerOptionsCloudinaryAnalyticsCustomData {
   customData3?: string;
   customData4?: string;
   customData5?: string;
+}
+
+export type AudioCodecType = string | "none" | "aac" | "vorbis" | "mp3";
```

Comment:
Should be able to, for the most part, copy in all of the types from the above int othe types/cloudinary-video-player.ts file

Isn't changes are correct?
According to issue need to copy from there!
Let me know how can I proceed here

Reply:
yeah apologies for the confusion, i thought we had decided that instead of copying in the type,s we would import it from the url-gen package, and i THINK that should be sufficient. so then you would point the video player transformation property to the imported type